examples/08-DeepAR-examples.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    train, val = day_series.split_after(pd.Timestamp("20170901"))
</pre><p>we can avoid scaling the day series as it's one-hot encoded</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>